lib/requests/foods.js

```diff
@@ -73,9 +73,27 @@ const renderFood = (food) => {
   )
 }
 
+const updateFood = (id) => {
+  var foodName = $(`.food-item-${id}`).children()[0].innerText
+  var foodCalories = $(`.food-item-${id}`).children()[1].innerText
```

Comment:
In continuation from the last comment, the reason I think that a user would only be able to edit the food chart from the foods.html page is because that functionality may yes, seem convenient, but may seem repetitive in the code so I'm wondering if its necessary.

Reply:
I think there are opportunities for refactoring (create a function for getFoodName that has a conditional based on the fileName, and then another one for getFoodCalories) if we decide that foods need to be editable on both pages.

Let's get thoughts from @corywest and @ameseee before proceeding!